packages/stylelint-config-terra/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@cerner/stylelint-config-terra",
+  "version": "3.7.0",
+  "description": "Terra Stylelint Shared Config",
+  "main": "stylelint.config.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/stylelint-config-terra"
+  },
+  "keywords": [
+    "stylelint",
+    "stylelint-config",
+    "Cerner",
+    "Terra",
+    "autoprefixer",
+    "css",
+    "scss",
+    "lint",
+    "linter"
+  ],
+  "engines": {
+    "node": "^10.0.0"
+  },
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-toolkit",
+  "files": [
+    "lib",
+    "stylelint.config.js"
+  ],
+  "scripts": {
+    "compile": "babel src --out-dir lib --copy-files --ignore src",
+    "lint": "eslint --ext .js,.jsx .",
+    "test": "tape lib/**/__tests__/**/*.js"
+  },
+  "dependencies": {
+    "@cerner/browserslist-config-terra": "3.1.0",
```

Comment:
Any issues here with the relative lerna structure/hoisting?

Reply:
No, it works fine in this package, lerna will link it to it's sibling package, i do need to unlock the version though...